httpclient-core/src/main/java/esa/httpclient/core/MultipartRequest.java

```diff
@@ -15,23 +15,67 @@
  */
 package esa.httpclient.core;
 
-import esa.commons.collection.MultiValueMap;
+import java.io.File;
+import java.util.Map;
+import java.util.function.Consumer;
 
-import java.util.List;
+public interface MultipartRequest extends ExecutableRequest {
 
-public interface MultipartRequest extends HttpRequest {
+    MultipartRequest multipartEncode(boolean multipartEncode);
 
-    /**
-     * Obtains attributes of current multipart body, which is a unmodifiable copy of current attributes.
-     *
-     * @return attributes
-     */
-    MultiValueMap<String, String> attributes();
+    boolean multipartEncode();
 
-    /**
-     * Obtains multipart files of current multipart body
-     *
-     * @return file items
-     */
-    List<MultipartFileItem> files();
+    MultipartRequest attribute(String name, String value);
```

Comment:
naming style shuld be the same, choose one of the `attr` and `attribute`

Reply:
done